test/unit/mocha-globals.js

```diff
@@ -17,11 +18,13 @@ beforeEach(() => {
 
   // set logger to log INFO, but do not append to console
```

Comment:
nit: I guess this comment is obsolete now? Maybe replace it with the link from a PR description.

Reply:
yes, thanks!